nozzleconfig/nozzle_config.go

```diff
@@ -76,6 +77,7 @@ func Parse(configPath string) (*NozzleConfig, error) {
 
 	overrideWithEnvUint32("NOZZLE_FLUSHDURATIONSECONDS", &config.FlushDurationSeconds)
 	overrideWithEnvUint32("NOZZLE_FLUSHMAXBYTES", &config.FlushMaxBytes)
+	overrideWithEnvInt("NOZZLE_GRAB_INTERVAL", &config.GrabInterval)
```

Comment:
Shouldn't it be `NOZZLE_GRABINTERVAL` like other variables above?

Reply:
To be honest, I think the other variable names are wrong. This way is much more legible, which I think is more important. It also correspond to the name of the property in the BOSH release: https://github.com/datadog/datadog-firehose-nozzle-release/blob/e56f24fb05725dde14c4221e0d9d6c5d93708c91/tile/tile.yml#L131

Reply:
Should we update all other variables then?

Reply:
Maybe in the next major release yeah, but not now as it will be breaking

Reply:
ok